Directory.Build.props

```diff
@@ -3,25 +3,20 @@
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
   <Import Project="$(RepositoryEngineeringDir)Compilers.props" />
 
-  <PropertyGroup Condition="'$(CopyrightNetFoundation)' != ''">
-    <Copyright>$(CopyrightNetFoundation)</Copyright>
```

Comment:
Do we also want to remove the Copyright? I only ask because that wasn't part of what @tmat suggested.

Reply:
per this: https://github.com/dotnet/arcade/pull/2003/files#diff-cdcd303385132649c3cafdaa56c23e01R11 it should be default now, right?

Reply:
No, keep the copyright.

Reply:
I only suggested removing the condition `Condition="'$(CopyrightNetFoundation)' != ''"`